nginx/conf.d/spider.conf

```diff
@@ -0,0 +1,29 @@
+map $http_user_agent $bad_client {
+    default 0;
+  ~*(360Spider)  1;
+  ~*(aiHitBot|AhrefsBot) 1;
+  ~*(betaBot|BlackWidow|Bolt|BLEXBot|BUbiNG) 1;
+  ~*(CazoodleBot|CPython|CCBot|ChinaClaw|Curious|CRAZYWEBCRAWLER|Custo) 1;
+  ~*(Default|DIIbot|DISCo|discobot) 1;
+  ~*(Exabot|eCatch|ecxi|EirGrabber|EmailCollector|EmailSiphon|EmailWolf|ExtractorPro|EyeNetIE) 1;
+  ~*(FlashGet|Findxbot) 1;
+  ~*(GetRight|GetWeb!|Go!Zilla|Go-Ahead-Got-It|Go.*package.*|GrabNet|Grafula|GT::WWW|GuzzleHttp) 1;
+  ~*(heritrix|HaosouSpider|HMView|HTTP::Lite|HTTrack) 1;
```

Comment:
this potentially can block some perl/python programs. Why do we need to response with 444 in this case? Also 444 have a special meaning "    Used to indicate that the server has returned no information to the client and closed the connection." Why do you want to use this code? It does not look like correct behavior.